fem/fespace.cpp

```diff
@@ -1558,6 +1556,21 @@ void FiniteElementSpace::Save(std::ostream &out) const
        << "FiniteElementCollection: " << fec->Name() << '\n'
        << "VDim: " << vdim << '\n'
        << "Ordering: " << ordering << '\n';
+
+   if (NURBSext)
+   {
+      if (NURBSext == mesh->NURBSext)
+      {
+         out << "NURBSext: " << 0 << '\n';
+      }
+      else
+      {
+         Array<int> Orders;
+         NURBSext->GetOrders(Orders);
+         out << "NURBSext: " << Orders.Size() << '\n';
```

Comment:
Remove before merging

Reply:
I might not understand what should be removed.
Let me explain what I tried to achieve.
When reading in an gridfunction, or more particular the associated fespace, it is necessary to be able to build the correct NURBSextension. For this two pieces of info are essential.

1) Does the fespace have its own NURBSextension, or does it use the one provided by the mesh. This is the isoparametric case. This is indicated with a NURBSext value of 0.

2)  If the fespace has its own NURBSextension the NURBSext value is non-zero. This can in fact be anything so it does not have to be the order size. The actual orders used in the constructor of the extension are specified next.

So I think we can not delete these statements. We could change Orders.Size() to 1 (or an other positive value). Or we need to change the mechanism of how the NURBSextension gets its knot vector associated orders.  Personally, I think the current mechanism is the most transparent I could come up with...